lms/djangoapps/instructor/tests/test_services.py

```diff
@@ -88,7 +89,8 @@ def test_reset_bad_content_id(self):
         result = self.service.delete_student_attempt(
             self.student.username,
             unicode(self.course.id),
-            'foo/bar/baz'
+            'foo/bar/baz',
+            requesting_user=self.student.username,
```

Comment:
Shouldn't this be a different user?  Also, not a username?  Is there an assertion that should fail for this?